10x/count/count.wdl

```diff
@@ -43,19 +46,21 @@ task setup_chunks {
     jq '.chunks' _outs > chunks.json
     jq '.chemistry_def' _outs > chemistry_def.json
     jq -r '.barcode_whitelist' _outs > barcode_whitelist.string
+
   >>>
   
   runtime {
     docker: "marcusczi/cellranger_clean:cromwell"
-    memory: "2 GB"
+    memory: "3.75 GB"
     cpu: 1
-    disks: "local-disk 1000 HDD"
+    disks: "local-disk 500 HDD" # must be >= input file size
```

Comment:
We might be better off making this a parameter to the workflow, instead of hard coding 500. It's also not clear to me why 500 is a reasonable number to use.